crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -682,7 +695,7 @@ pub(crate) async fn dialog_export_selection_w_prefs(appwindow: &RnAppWindow, can
     export_file_button.connect_clicked(
         clone!(@strong selected_file, @weak export_file_label, @weak button_confirm, @weak dialog, @weak canvas, @weak appwindow => move |_| {
             glib::spawn_future_local(clone!(@strong selected_file, @weak export_file_label, @weak button_confirm, @weak dialog, @weak canvas, @weak appwindow => async move {
-                dialog.hide();
+                dialog.set_sensitive(false);
```

Comment:
Shouldn't this be [set_visible](https://gtk-rs.org/gtk4-rs/stable/0.8/docs/gtk4/prelude/trait.WidgetExt.html#method.set_visible) ?

Reply:
From a UX standpoint in this step we are still interacting with the dialog so it shouldn't be hidden. Also even if we did hide it we can't interact with the canvas anyway (we will have a non-intractive main window without an indication).

By having `sensitive` set to `false` we have an indication that we can't interact with the dialog or the canvas while we are selecting a file.


It might make since with the old `gtk::Dialog` since it was a separate window (it was like we did replace the dialog with a file-selection window). But with `adw::Dialog` it is bad for UX.

Reply:
IIRC it was an issue when the modal gtk dialog had the file picker stacked on top of it. But good point, adw::Dialog is not a separate Window anymore